src/index.ts

```diff
@@ -189,8 +190,45 @@ export const Rune: RuneExport = {
     // Reset randomness to be deterministic across plays
     Rune.deterministicRandom = Rune._randomNumberGenerator(Rune.getChallengeNumber())
   },
+  _getQueryParams: () => {
+    return decodeURI(window.location.search)
+        .replace('?', '')
+        .split('&')
+        .map(param => param.split('='))
+        .reduce((values, [ key, value ]) => {
+          values[ key ] = value
+          return values
+        }, {} as {[key: string]: string})
+  }
 }
 
+//Safari ios throttles requestAnimationFrame when user has not interacted with the iframe at least once.
+//In case the games are not using clicks (for instance only swiping), ios will not treat these interactions
+//with the iframe as user interacting. As a workaround, in the browser we will start overlay with
+//click events disabled and display an invisible div inside the iframe above the canvas.
+//This way the users will click on the transparent div element the very first time. We will let our client
+//know about it with _INITIAL_OVERLAY_CLICK event and the transparent div will remove itself.
+//Afterwards the play/pause will be once again fully controlled by our client.
+;(() => {
+  const queryParams = Rune._getQueryParams()
+
+  if (!!queryParams.webBrowser && queryParams.webBrowser === '1') {
```

Comment:
Maybe instead of using webBrowser we could use `enableInitialOverlay`? That would feel a bit more specific.

Reply:
👍🏽 Maybe even `enableInitialOverlayInBrowser` to make both things clear (only enabled in browser and what is being enabled)